docs/tools/IDE/updating_firmware.md

```diff
@@ -2,46 +2,50 @@
+<span class="tips">**Tip**: The fastest way to learn about firmware updates in the Online Compiler is to try the [Quick Connect guide](../guides/connect-device-to-pelion) to create the original application, and the [Quick Update guide](../guides/pelion-firmware-update) to perform a firmware update.</span>
 
 <span class="notes">You need a [Pelion Device Management](https://portal.mbedcloud.com/login) account to use the update workflow.</span>
 
+<!---
 ## Creating an application you can update
 
 To be able to receive updates, an application must have:
 
-- A **connect certificate** to access a Pelion Device Management account. When you run the application on your device, the bootstrap server will use this connect certificate to authenticate your application with your Pelion Device Management account.
+- A **connect certificate** to access a Pelion Device Management account. When you run the application on your device, the bootstrap server will use this connect certificate to authenticate your application with your Pelion Device Management account. <!---We don't use 'connect certificate' anywhere else. Seems it might come from the online compiler.---><!---
 - An **update certificate** to verify that incoming updates come from a trusted source.
 - A **bootloader** to verify and install update images.
-- The Device Management Client, [normally imported as part of the example application](https://github.com/ARMmbed/mbed-cloud-client-example). You can also import it using the [Quick Connect guide we mentioned above](https://cloud.mbed.com/guides/connect-device-to-pelion).
+- Device Management Client, [normally imported as part of the example application](https://github.com/ARMmbed/mbed-cloud-client-example). You can also import it using the [Quick Connect guide we mentioned above](https://cloud.mbed.com/guides/connect-device-to-pelion).
 
 You need:
 
-- A **private key** so you can sign the firmware update manifest file.
-- A **manifest file**, which defines the update, including the location of the new firmware image and the type of device the update applies to. The manifest file is signed with the private key to assure the device that the image is from a trusted source and has not been tampered with.
+- A **private key** so you can sign the firmware update manifest file. <!---check vocabulary---><!---
+- A **manifest file**, which defines the update, including the location of the new firmware image and the type of device the update applies to. The manifest file is signed with the private key to assure the device that the image is from a trusted source and has not been tampered with. <!---We often just call it a manifest, not a manifest file.---><!---
 
 For development use cases only, you can use the Online Compiler to generate the certificates and the manifest file. For production use cases, you would use offline tools to generate and store them securely. Learn more [about the developer workflow below](#notes-about-the-developer-workflow).
```

Comment:
On second thought, this should probably stay in. Though the Update docs *do* make this quite clear.